rpc_jobs/rpc_openstack.yml

```diff
@@ -634,3 +634,17 @@
           run_issue_linker: true
     jobs:
       - 'Pull-Request-Whisperer_{repo}'
+
+- project:
+    name: 'rpc-openstack-master-checkmarx'
+    jira_project_key: "RI"
+    trigger:
+      - RI
+    scan_type:
+      - pci
```

Comment:
Would you be interested in the all scan as well? It creates a separate job so that PCI progress can be tracked separately.

Reply:
Added